pkg/cmd/server/start/start_master.go

```diff
@@ -609,6 +609,9 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 	// Start these first, because they provide credentials for other controllers' clients
 	oc.RunServiceAccountsController()
 	oc.RunServiceAccountTokensController(controllerManagerOptions)
+	// Must start a few informers now
+	oc.Informers.KubernetesInformers().Start(utilwait.NeverStop)
```

Comment:
These need to start to make the SA token controller active, right?  Be more descriptive in your comment.  Lots of people end up here.